src/ViewModels/CkanGroupViewModel.js

```diff
@@ -76,6 +77,14 @@ var CkanGroupViewModel = function(application) {
      */
     this.minimumMaxScaleDenominator = undefined;
 
+    /**
+     * Gets or sets any extra wms parameters that should be added to the wms query urls in this CKAN group.
+     * If this property is undefined or if {@link CkanGroupViewModel#filterByWmsGetCapabilities} then no parameters are added
```

Comment:
I'm pretty sure this sentence isn't true.  `wmsParameters` is passed to the created WMS catalog items as their `parameters` property, regardless of the `filterByWmsGetCapabilities` property, correct?